site/gatsby-site/src/utils/date.js

```diff
@@ -9,3 +11,14 @@ export const validateDate = (date) => {
   }
   return date;
 };
+
+export const isPastDate = {
+  message: '*Date must be in the past',
```

Comment:
Looks like the translation is missing here, but seems to be missing for other error messages too